src/modules/menu/components/toolboxRight/CompassButton.vue

```diff
@@ -23,19 +26,33 @@
 </template>
 
 <script>
-import { mapActions, mapState } from 'vuex'
+import { mapActions } from 'vuex'
 export default {
+    inject: ['getMap'],
     emits: ['northen'],
-    computed: {
-        ...mapState({
-            rotation: (state) => state.position.rotation,
-        }),
+
+    data() {
+        return {
+            rotation_: 0,
```

Comment:
Is the underscoring meaning it is a private prop?
If so I prefer a prefixed underscoring, so you don't stumble upon a private prop with autocompletion by accident, you have to actively write first a underscore to list them

Reply:
I removed the underscore entirely, as we don't use this convention anywhere else in the code.